src/main/resources/META-INF/plugin.xml

```diff
@@ -134,6 +134,24 @@
                 <override-text place="GitMacheteToolbar"/>
             </action>
 
```

Comment:
I'd also add `Slide In Under Current` action to the toolbar... right-clicking on the current branch just to add a branch is kinda inconvenient